config/bundles.php

```diff
@@ -43,4 +43,5 @@
     Symfony\Cmf\Bundle\RoutingBundle\CmfRoutingBundle::class => ['all' => true, 'website' => true],
     Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle::class => ['all' => true],
     Sulu\Bundle\ActivityBundle\SuluActivityBundle::class => ['all' => true],
+    Sulu\Bundle\TrashBundle\SuluTrashBundle::class => ['all' => true],
```

Comment:
I personally would not activate the trash bundle by default, would like to keep the skeleton slim and not bloated with everything. /cc @chirimoya

Reply:
`Trash` is for me more like the `ActivityBundle` and not a "add-on" which developers should activate or add.

Reply:
> Trash is for me more like the ActivityBundle and not a "add-on" which developers should activate or add.

When saving of activites is disabled also that tabs and menu items should not appear as they are not showing anything then. Currently ActivityBundle is from my side just required as the core concept of the Events are currently living in there, which was already at somepoint in discussion to move out of the activity bundle.
I'm worried if we continue adding things like this to the skeleton we ending up with a lot of menu points like other CMSs which UI feels flooded with too much and so have a bad impact on UX.